test/workflows/components/workflows-v1alpha1.libsonnet

```diff
@@ -267,14 +264,6 @@
                     name: "unit-test",
```

Comment:
unit tests not changed in v1alpha1?

Reply:
We cannot change it, because v1alpha1 unit test should be run after the cluster is created.